### DISCUSSION TITLE
cannot install from source, ./configure script not found ?

### DISCUSSION THREAD

**Discussion by DennisD2** [libsidplayfp/libsidplayfp#99] [Q&A]
According to INSTALL, I tried to install. But there is no "configure" script.

mypc:src> unzip ~/Downloads/sidplayfp-master.zip 
Archive:  /home/dennis/Downloads/sidplayfp-master.zip
...
mypc:src/sidplayfp-master> ls
AUTHORS  build-aux  configure.ac  COPYING  doc  INSTALL  m4  Makefile.am  README  src  TODO
mypc:src/sidplayfp-master> ./configure
bash: ./configure: File not found

Is there a file missing or another step needed before calling configure?

**Comment by mywave82** [comment]
README states inside one of the paragraphs

`autoreconf -i`

**Reply by DennisD2** [reply]
With this before the "configure; make; make install" pipeline, all worked well, thanks.
In libsidplay README file, this initial step is explained somewhat better than in sidplayfd INSTALL file.
